java/java.mx.project/src/org/netbeans/modules/java/mx/project/SuiteClassPathProvider.java

```diff
@@ -76,7 +61,7 @@ public ClassPath findClassPath(FileObject file, String type) {
 
     @Override
     public void projectOpened() {
-        GlobalPathRegistry.getDefault().register(ClassPath.BOOT, new ClassPath[] {bootCP});
+        // GlobalPathRegistry.getDefault().register(ClassPath.BOOT, new ClassPath[] {bootCP});
```

Comment:
Why the BOOT cp is not registered and unregistered?

Reply:
Now I would have to collect all bootpaths from all the used platforms.

Reply:
I think it can work. The boot cp should be indexed because it's discovered in the `RepositoryUpdater#findDependencies(...)`. The `PathRegistry#getRootPathIds()` will not know the boot cp roots but `PathRegistry#getRootPathIds()` is not important for binary roots.
Just check that the boot cp is indexed (you see the platform types from the correct platform in Go To Type dialog).

Reply:
Done in 3c0dfd659abeabadbd5f33b511f5a5151cfe6512